packages/sw-build-webpack-plugin/index.js

```diff
@@ -6,17 +6,25 @@ const swBuild = require('sw-build');
  * const SwBuildWebpackPlugin = require('sw-build-webpack-plugin');
  * .
  * .
- * plugins: [
- * 	new SwBuildWebpackPlugin({
- * 		rootDirectory: './build/',
- * 		dest: './build/sw.js',
- * 		globPatterns: ['**\/*.{html,js,css}'],
- * 		globIgnores: ['admin.html'],
- * 		templatedUrls: {
- * 			'/shell': ['shell.hbs', 'main.css', 'shell.css'],
- * 		},
- * 	});
- * ]
+ * module.exports = {
+ *   entry: {
+ *     app: './step1/app.js'
+ *   },
+ *   output: {
+ *     path: __dirname + '/step1/public/js',
+ *     publicPath: '/public/js/',
+ *     filename: '[name].js',
+ *   },
+ *   plugins: [
+ *   	new SwBuildWebpackPlugin({
+ *   		globDirectory: './build/',
```

Comment:
The indentation is weird here, as it is in a lot of your files—are you using tabs instead of spaces? Normally it wouldn't matter much, but this will be user-facing.